app/src/main/java/com/duckduckgo/app/bookmarks/ui/BookmarksActivity.kt

```diff
@@ -87,8 +88,9 @@ class BookmarksActivity: DuckDuckGoActivity() {
         finish()
     }
 
+    @Suppress("deprecation")
     private fun confirmDeleteBookmark(bookmark: BookmarkEntity) {
-        val message = getString(R.string.bookmarkDeleteConfirmMessage, bookmark.title)
+        val message = Html.fromHtml(getString(R.string.bookmarkDeleteConfirmMessage, bookmark.title))
         val title = getString(R.string.bookmarkDeleteConfirmTitle)
         alert(message, title) {
```

Comment:
`Activity` gets leaked if rotated while showing `confirm deletion` prompt.

Reply:
Nice catch! Happy to be have that cleaned up for the next release.